### PR TITLE
Conference

### PR DESCRIPTION
**Pull request by marcoacierno**
Base PR for the conference features

"Sub tasks" PRs:

- CFP: #374
- Voting
- Tickets

### CODE REVIEW

**Comment by marcoacierno** [comment]
Let me know what do you think so far of the code @patrick91  :)

File: backend/languages/languages.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
...
+'''
+
+LANGUAGES = json.loads(_codes)
```
Comment (patrick91) [inline backend/languages/languages.py]: I'd add just a few language for now, like Italian and English, no need to populate all the languages of the world :)

Comment (marcoacierno) [reply backend/languages/languages.py]: done! :)

**Comment by marcoacierno** [comment]
@patrick91 we have the `Ticket` model here, but do we need it considering we will using ti.to? To list the tickets in the frontend we can implement it fetching the tickets from ti.to (and caching them to avoid problems)

**Pull request merged**